warpdrive.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+#
+# the warp drive allows you to enter hyperspace
+
+cd "$(dirname "$0")"
+
+DOCKER="docker"
```

Comment:
prefer `: ${DOCKER:=sudo docker}`, and then folks can just run with `DOCKER=podman ./hyperspace ...` if they don't use docker

Reply:
Is that basically `DOCKER=${DOCKER:-sudo docker}`?